service/common/src/main/java/org/apache/polaris/service/catalog/iceberg/IcebergCatalog.java

```diff
@@ -1577,7 +1605,7 @@ private String newTableMetadataFilePath(TableMetadata meta, int newVersion) {
    * of this code was originally copied from {@link org.apache.iceberg.view.BaseViewOperations}.
    * CODE_COPIED_TO_POLARIS From Apache Iceberg Version: 1.8
    */
-  private class BasePolarisViewOperations extends PolarisOperationsBase<ViewMetadata>
+  public class BasePolarisViewOperations extends PolarisOperationsBase<ViewMetadata>
```

Comment:
This is public just for parity with `BasePolarisTableOperations`, I can make it private. `BaseMetastoreOpertations` was a public type though, so it does seem reasonable just to have them be public